src/main/java/io/camunda/testing/extensions/ZeebeAssertionsExtension.java

```diff
@@ -49,8 +49,8 @@ private Field getRecordStreamField(final ExtensionContext extensionContext) {
     return recordStreamOptional.orElseThrow(
         () ->
             new IllegalStateException(
-                "No RecordStreamSource has been found. Please make sure a "
-                    + "RecordStreamSource field has been declared in the test class."));
+                "Expected a field of type RecordStreamSource to be declared in the test class, "
+                    + "but none has been found"));
```

Comment:
🔧 To be honest, now I miss the user hint. Would be nice to add it behind it.

Something like:
```
"Expected a field of type RecordStreamSource to be declared in the test class, "
    + "but none has been found. Please make sure a field of type RecordStreamSource "
    + "has been declared in the test class."
```